include/net/lwm2m.h

```diff
@@ -136,6 +136,11 @@ struct lwm2m_ctx {
 	 *  copied into the actual resource buffer.
 	 */
 	uint8_t validate_buf[CONFIG_LWM2M_ENGINE_VALIDATION_BUFFER_SIZE];
+
+	/**
+	 * Current FW update object instance.
+	 */
+	uint16_t fw_obj_inst;
```

Comment:
I'm not sure if this variable belongs in a generic lwm2m_context structure. Anyway, it shall not be needed with the `lwm2m_pull_context` from the upstream PR.